app/views/admin/applications/show.html.erb

```diff
@@ -0,0 +1,67 @@
+<h1><%= @application.name %></h1>
+
+<h3>Street Address: <%= @application.street_address %></h3>
+<h3>City: <%= @application.city %></h3>
+<h3>State: <%= @application.state %></h3>
+<h3>Zip Code: <%= @application.zip_code %></h3>
+<h3>Reason for applying: <%= @application.description %></h3>
+<h3>Application Status: <%= @application.status %></h3>
+
+<h3>Applying for pets: </h3>
+<ul>
+    <% @application.pets.each do |pet| %>
+        <li><p><%= link_to "#{pet.name}", "/pets/#{pet.id}" %></p></li>
+        <% if ApplicationPet.where({pet_id: pet.id,application_id: @application.id}).first.status == "Approved" %>
```

Comment:
We don't want to use `.where` in the view as that is an AR method that triggers a database query. Per MVC, this would better off being encapsulated in a model method.